src/index.js

```diff
@@ -0,0 +1,196 @@
+import _, { add, times } from 'lodash';
+import { clearAll } from './script.js';
+import taskCompleted from './script.js';
+import './style.css';
+
+const tasks = [
+	{
+		description: 'Go to the GYM',
+		completed: true,
+		index: 0
+	},
+	{
+		description: 'Read ES6 book',
+		completed: true,
+		index: 1
+	},
+	{
+		description: 'Go shopping: ',
+		completed: true,
+		index: 2
+	},
+	{
+		description: 'Wash clothes',
+		completed: true,
+		index: 3
+	},
+	{
+		description: 'Add New Feature to the app',
+		completed: true,
+		index: 4
+	},
+	{
+		description: 'Go camping',
+		completed: true,
+		index: 5
+	},
+	{
+		description: 'Go to church',
+		completed: true,
+		index: 6
+	},
+	{
+		description: 'Cook lunch',
+		completed: true,
+		index: 7
+	},
+	{
+		description: 'Go out',
+		completed: true,
+		index: 8
+	}
+
+]
+
+const myContainer = document.getElementById('to-do-list');
+tasks.forEach((item) => {
+	myContainer.innerHTML += `
+	<ul class='list-container'>
+		<li class='list-items'>
+		<input type="checkbox" id="check">
+		${item.description}
+		</li>
+	</ul>
+`;
+});
+
+myContainer.innerHTML += `
+<div class='footer-list'>
+	<a href='#' class='footer-a'>Clear all completed</a>
+</div>
+`;
+
+
+const formContainer = document.getElementById('form')
+const inputElement = document.getElementById('input-element');
+const myContainer = document.getElementById('items-container');
```

Comment:
- [ ] This error occurs when the page loads, and I believe it's got to do with the fact that `myContainer` has been declared previously on line `55`. Please see error 🐛 below and try to fix it 👇 
![image](https://user-images.githubusercontent.com/46839548/194700625-01ae36d7-e1ab-4bf2-a490-0dfb4a9546eb.png)